templates/default/slurm/slurmd.service.erb

```diff
@@ -1,12 +1,13 @@
 [Unit]
 Description=Slurm node daemon
-After=munge.service network.target remote-fs.target
+After=munge.service network-online.target remote-fs.target
+Wants=network-online.target
 ConditionPathExists=/opt/slurm/etc/slurm.conf
 
 [Service]
 Type=simple
 EnvironmentFile=-/etc/sysconfig/slurmd
-ExecStart=/opt/slurm/sbin/slurmd -D $SLURMD_OPTIONS
+ExecStart=<%= node['cfncluster']['slurm']['install_dir'] %>/sbin/slurmd -D -s $SLURMD_OPTIONS
```

Comment:
I just wanted to clarify. Since the template states the /slurm/compute directory. Would that mean that I need to move the file slurmd.service to, and create if necessary, the /slurm/compute directory? Also, why would I create a compute_node folder?

Reply:
Yes, please create a `compute` folder. My bad to refer it to `compute_node`